src/acp/matcher.test.ts

```diff
@@ -1320,51 +1326,21 @@ describe("getClientAll", () => {
   });
 });
 
-describe("setClient", () => {
```

Comment:
I would not remove the tests until the deprecated function is removed from the codebase, instead of removing the implementation from the code coverage.